lib/bigquery/index.js

```diff
@@ -96,15 +96,10 @@ var SCOPES = ['https://www.googleapis.com/auth/bigquery'];
  */
 function BigQuery(options) {
   if (!(this instanceof BigQuery)) {
+    options = util.normalizeArguments(this, options);
```

Comment:
I can do that, I just want to point out that doing that will call normalizeArguments twice per object creation.